web_widget_callto/static/src/js/widget_callto.js

```diff
@@ -0,0 +1,31 @@
+odoo.define('web_widget_callto.WidgetCalltoCustom', function(require) {
+"use strict";
+
+var field_registry = require('web.field_registry');
+var basic_fields = require('web.basic_fields');
+
+var WidgetCalltoCustom = basic_fields.FieldEmail.extend({
+    prefix: 'callto',
```

Comment:
`init`, `getFocusableElement`, `_renderReadonly` and `supportedFieldTypes` is not needed anymore.

Reply:
True, (As we are using `FieldEmail` now)
Fixed and Pushed.